website/docs/using-odo/create-component.md

```diff
@@ -3,4 +3,139 @@ title: Create Component
 sidebar_position: 1
 ---
 # Creating components using odo
-This doc explains all the different ways and possibilities for a user to run `odo component create`
\ No newline at end of file
+
+[Component](../getting-started/basics#component) is the most basic unit of operation for odo. And the way to create one is using `odo create` (short for `odo component create`) command.
+
+In simplest terms, when you "create" an odo component, you populate your current working directory with the file `devfile.yaml`. It is a manifest file that contains information about various resources (URL, Storage, Services, etc.) that correspond to your component, and will be created on the Kubernetes cluster when you execute `odo push` command. Most odo commands will first modify (add or remove configuration from) this file, and then subsequent `odo push` will create or delete the resources from the Kubernetes cluster. However, odo users are not expected to know how the `devfile.yaml` is organized; it is the odo commands that would create, update, or delete it.
+
+In this guide, we are going to create a Spring Boot component to deploy the [Spring Boot petclinic](https://github.com/spring-projects/spring-petclinic) project to a Kubernetes cluster.
+
+Let's clone the project first:
+```shell
+git clone https://github.com/spring-projects/spring-petclinic
+cd spring-petclinic
+```
+
+Next, let's create a project <!-- add link to project command reference here --> on the Kubernetes cluster in which we will be creating our component:
+```shell
+odo project create myproject
+```
+Alternatively, you could also use one of the existing projects on the cluster:
+```shell
+odo project list
+odo project set <project-name>  # replace <project-name> with a valid value from the list
```

Comment:
Copying the whole block is not helpful. It would be better if we separate these 2 commands into different codeblocks.